package.json

```diff
@@ -27,9 +28,11 @@
     "lint": "eslint modules"
   },
   "dependencies": {
+    "@types/history": "^2.0.41",
+    "@types/react": "^0.14.52",
```

Comment:
These dependencies are just typings and don't affect anything to the compilation steps or production code. The moment project does the same. We can make them dev dependencies too. The problem with the last is that when a user installs the react-router library it will have to guess that you have a dependency for history and installs the typings for that too, which is not good too.